Source/WebCore/Modules/mediasource/SourceBuffer.cpp

```diff
@@ -1704,15 +1706,18 @@ void SourceBuffer::sourceBufferPrivateDidReceiveSample(SourceBufferPrivate*, Med
         if (m_shouldGenerateTimestamps)
             m_timestampOffset = frameEndTimestamp;
 
+        if (trackBuffer.fudgeRoom.isInvalid() || trackBuffer.fudgeRoom < frameDuration)
+            trackBuffer.fudgeRoom = frameDuration;
```

Comment:
current fudgeFactor equals to 2*frameDuration for a purpose (https://bugs.webkit.org/show_bug.cgi?id=159813). Please increase.